homeassistant/components/netdata/sensor.py

```diff
@@ -145,6 +147,72 @@ async def async_update(self):
         )
 
 
+class NetdataAlarms(Entity):
+    """Implementation of a Netdata alarm sensor."""
+
+    def __init__(self, netdata, name, host, port):
+        """Initialize the Netdata alarm sensor."""
+        self.netdata = netdata
+        self._state = None
+        self._name = name
+        self._host = host
+        self._port = port
+
+    @property
+    def name(self):
+        """Return the name of the sensor."""
+        return f"{self._name} Alarms"
+
+    @property
+    def state(self):
+        """Return the state of the resources."""
+        return self._state
+
+    @property
+    def device_info(self) -> Optional[Dict[str, Any]]:
```

Comment:
`device_info` is only active for integrations using config entries. Please remove this for now.

Reply:
Removed.